### PR TITLE
use local path for examples

### PR DESCRIPTION
**Pull request by wilzbach**
@9il I discovered that `dub` supports the `path` option to supply local dependencies -> let's clean our `dub.json`

### CODE REVIEW

File: examples/lda_hoffman_sparse.d [hunk 1]
```diff
@@ -26,7 +26,7 @@ void main(string[] args)
     if (args.length > 1)
         curFolder = args[1];
     else
-        curFolder = thisExePath.dirName;
+        curFolder = getcwd.buildPath("examples");
```
Comment (wilzbach) [inline examples/lda_hoffman_sparse.d:29]: temporary build are done in `/tmp/...`, hence I guess `getcwd` is the best we could try?

**Comment by wilzbach** [comment]
@9il It seems like I accidentally bumped into a `dub` bug - https://github.com/dlang/dub/issues/893

I added the following workaround to our builds:

```
dub fetch imageformats
dub build imageformats
```

It's not a perfect solution, but I hope that the `dub` bug gets resolved soon, s.t. we can remove this ;-)

**Comment by 9il** [comment]
fails

**Comment by wilzbach** [comment]
> fails

passing (:

**Pull request merged**